src/applications/vaos/tests/services/clinic/index.unit.spec.js

```diff
@@ -0,0 +1,61 @@
+import { expect } from 'chai';
+import {
+  mockFetch,
+  setFetchJSONResponse,
+  setFetchJSONFailure,
+  mockMultipleApiRequests,
+} from 'platform/testing/unit/helpers';
+
+import { getLocation } from '../../../services/location';
+import { getAvailableHealthCareSystems } from '../../../services/clinics';
```

Comment:
It looks like there are still files from a `services/clinics` folder in this PR, I think those should be deleted?

Reply:
Done.